spec/requests/authentication_spec.rb

```diff
@@ -55,6 +55,16 @@
       }
       expect(response.parsed_body).to include(expected)
     end
+
+    it "returns a correctly formatted collection hrefs" do
+      api_basic_authorize
+
+      get api_entrypoint_url
+
+      response.parsed_body["collections"].each do |collection|
```

Comment:
WDYT of doing something like this for the body here:

```ruby
      collection_names = Api::ApiConfig.collections.to_h.select { |_, v| v.options.include?(:collection) }.keys
      hrefs = collection_names.collect { |name| url_for(:controller => name, :action => "index") }
      expected = {
        "collections" => a_collection_containing_exactly(
          *hrefs.collect { |href| a_hash_including("href" => href) }
        )
      }
      expect(response.parsed_body).to include(expected)
```

Aithough there's a bit more going on there, I like it because:

1. It verifies against an external source of truth (if for instance there were no collections in your response, you'd still pass)
2. It looks for an exact match on the href
3. It's a single expectation, which will give you one failure and a diff in the event of a regression